serialization/protobuf-serializer/src/main/java/Member.java

```diff
@@ -0,0 +1,48 @@
+import com.hazelcast.config.Config;
+import com.hazelcast.config.SerializerConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.query.SqlPredicate;
+
+import static com.hazelcast.core.Hazelcast.newHazelcastInstance;
+import static com.hazelcast.query.Predicates.equal;
+import static com.hazelcast.serialization.PersonProtos.Person;
+
+/**
+ * Hazelcast Member with custom serialization registration
+ *
+ * @author Viktor Gamov on 5/12/17.
+ *         Twitter: @gamussa
+ */
+public class Member {
+    public static void main(String[] args) {
+        Config config = new Config();
+
+        // configure hazelcast serializers to use provided serializer for protobuf
+        final SerializerConfig personProtoSerializerConfig = new SerializerConfig();
+        personProtoSerializerConfig.setTypeClass(Person.class);
+        personProtoSerializerConfig.setImplementation(new PersonProtoSerializer());
+        config.getSerializationConfig().addSerializerConfig(personProtoSerializerConfig);
+        // end serialization config
+
+        final HazelcastInstance hazelcastInstance = newHazelcastInstance(config);
+        final IMap<Integer, Person> personsMap = hazelcastInstance.getMap("person");
+        populatePerson(personsMap);
```

Comment:
I would call this just 'populate'. Persons is already implied by the map name and populatePerson sounds like a single person being added.